lib/views/auth/widgets/social_media_auth.dart

```diff
@@ -1,21 +1,17 @@
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 
-// Widgets
-import 'login_signup_widgets.dart';
 import 'social_media_button.dart';
 
 class SocialMediaAuth extends StatelessWidget {
-  final VoidCallback onAnonymousSignIn;

```

Comment:
No more anonymous signin? 🤔

Reply:
yeaa, i feel like it would affect future model if users just sign up for anonymous account if we were to limit the number of pictures taken to 5 for free version and unlimited for pay version if we decide to go that route.